src/shared/rotation_op.c

```diff
@@ -0,0 +1,604 @@
+/* Copyright (C) 2015-2019, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is a free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation
+ */
+
+#include "shared.h"
+
+static const char * MONTHS[] = {
```

Comment:
This is repeated several times in the code. Could you move it to a common file?

Reply:
It's not possible due to crossed dependencies. A rework of significant Wazuh code would be needed in order to achieve this.